lessons/Data-Science/lookups/langs/en-us/index.adoc

```diff
@@ -122,31 +90,88 @@ It's often better to name our Row definitions according to the _property we care
+We can access _specific columns_ of a Row, by using a *Row Accessor*.
+
+- start with a row value:
+  ** a `row-n` expression just like the ones we've been working with
+  ** the name of a row that we have already defined
+- end with square brackets `[ ]` containing the name of the column where the value can be found.
+
+Here are three examples that use row accessors to get at different columns from the first row in the animals-table.
+
+@indented{
+[cols="1a,1a", options="header"]
+|===
+| Without Predefined Rows
+| Using Predefined Rows
+
+|
 
 ```
 row-n(animals-table, 0)["name"]
 row-n(animals-table, 0)["age"]
 row-n(animals-table, 0)["fixed"]
 ```
 
-And of course, we can use our defined name, substituting it in place of all the redundant code:
+|
 
 ```
 cat-row["name"]
 cat-row["age"]
 cat-row["fixed"]
 ```
 
+|===
+}
+
 === Investigate
 
+Sometimes we want to ask questions about a specific Row, or do some computations with that Row:
+
+- Is the age of our `lizard-row` animal more than 3?
+- How many _kilograms_ does our `rabbit-row` weigh?
+- Is the species of our `fixed-row` animal a `"dog"`?
+- Could we use the `age` of our `male-row` as the size of a solid, green triangle?
+
+We can use our knowledge of lookups, operators and functions to do just that!
+
+@lesson-instruction{
+- Turn to @printable-exercise{lookup-questions-matching.adoc}.
+- Notice the `pet-table` at the top of the page. All of the questions on this page will refer back to that table.
+- Complete the page, discussing your thinking with a partner.
+}
+
+@teacher{Confirm that students have correctly complete the page before moving on. If they need more matching practice before moving on have them complete @opt-printable-exercise{more-practice-w-lookups.adoc}.}
+
+Now that we know how to lookup a value for a specific column of a row, let's practice some of the things we might be able to do with that information!
+
+@teacher{Students will need to have all of the values from @printable-exercise{defining-rows} defined in the Definitions Area in order to work through this next set of exercises.}
+
 @lesson-instruction{
-- Complete the exercises on @printable-exercise{lookup-questions-matching.adoc}.
-- Complete @printable-exercise{more-practice-w-lookups.adoc}
+- Open your copy of the @starter-file{animals} and make sure the Definitions Area includes defintions for all of the rows from @printable-exercise{defining-rows}.
+- Turn to @printable-exercise{lookup-expressions.adoc} and notice that the code at the top uses the "age" of `old-row` to determine the size of the triangle.
+- Type the code into the Interactions Area and test it out.
+- With a partner, work through @printable-exercise{lookup-expressions.adoc}
+}
+
+@teacher{
+Have students share their most creative uses of lookups. Did anyone use three or more columns from a Row? Did anyone use more than one Row in the same expression?
 }
 
 === Synthesize
```

Comment:
@schanzer This is a pretty unorthodox Synthesize section in that it looks forward rather than backward. I'm not totally sure that it works. They don't actually know what a Table Function is yet and they're being asked to reason about them.